pages/_app.tsx

```diff
@@ -50,12 +51,14 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
 
       <QueryClientProvider client={queryClient}>
         <AuthProvider>
-          <Hydrate state={pageProps.dehydratedState}>
-            <Head />
-            {getLayout(<Component {...pageProps} />)}
-            <ToastContainer theme="dark" />
-            <ReactQueryDevtools initialIsOpen={false}></ReactQueryDevtools>
-          </Hydrate>
+          <SharedGuidesProvider>
```

Comment:
If we do end up going with this context, lets not wrap at the app root but rather the ballot page

Reply:
I'm removing the context